dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/bootstrap/DubboBootstrap.java

```diff
@@ -1080,23 +1083,33 @@ private void exportServices() {
             if (exportAsync) {
                 ExecutorService executor = executorRepository.getServiceExporterExecutor();
                 Future<?> future = executor.submit(() -> {
-                	try {
-                        sc.export();
-                        exportedServices.add(sc);
-                	}catch (Throwable t) {
-                		logger.error("export async catch error : " + t.getMessage(), t);
-					}
+                  try{
+                    exportService(serviceConfig);
+                  }catch(Throwable t){
+                    logger.error("export async catch error : " + t.getMessage(), t);
+                  }
                 });
                 asyncExportingFutures.add(future);
             } else {
-                sc.export();
-                exportedServices.add(sc);
+                exportService(serviceConfig);
             }
         });
     }
 
+    private void exportService(ServiceConfig sc){
+        if (exportedServiceNames.contains(sc.getServiceName())) {
+            throw new IllegalArgumentException("The Duplicated BeanDefinition of ServiceBean[ServiceConfig:" +
```

Comment:
Using IllegalStateException is better than IllegalArgumentException here.

Reply:
Need to print duplicate ServiceName.
It would be better if duplicate instances of ServiceConfig could be printed.

Reply:
Need to explain the inspection rules, triples (group, interface, version) cannot be repeated. Users who see errors need to know what's going on and how to fix them.

Reply:
ok , all changes pushed, could you pls help me to fix test case.
https://github.com/apache/dubbo/issues/7623